components/memcached.libsonnet

```diff
@@ -0,0 +1,158 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
```

Comment:
it would be nice to combine this with the memcached usage in telemeter, i.e. use this component directly in the telemeter configuration that way we have less duplication of jsonnet config. We can do this as a follow up

Reply:
This is a good idea, I'll address this in another PR.

Reply:
https://github.com/observatorium/configuration/issues/243